blocks/product-gallery-block/features/product-gallery/default.jsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useFusionContext, useComponentContext } from "fusion:context";
+import { RESIZER_APP_VERSION, RESIZER_URL } from "fusion:environment";
+import getProperties from "fusion:properties";
+import getTranslatedPhrases from "fusion:intl";
+
+import { Carousel, Icon, Image } from "@wpmedia/arc-themes-components";
+
+const BLOCK_CLASS_NAME = "b-product-gallery";
+
+export function ProductGalleryDisplay(props) {
+	const { arcSite, data, id, isFeaturedImageEnabled, resizerAppVersion, resizerURL } = props;
+
+	const { locale } = getProperties(arcSite);
+	const phrases = getTranslatedPhrases(locale);
+	const carouselItems = data?.schema?.productGallery?.value?.assets
```

Comment:
Two items 

1. Should this be done ahead of the presentational component? - That way the current `data` prop doesn't need to know about the nested object location
2. If there is no items we should return `null`

Reply:
cool will do! thanks

Reply:
updated @matthewroach